controllers/constants.go

```diff
@@ -49,7 +49,7 @@ const (
 	// cconf and hconf
 	confExploreEnabled        = "explore.enabled"
 	confLocalDataDirKey       = "local.data.dir"
-	confLocalDataDirVal       = "/data"
+	confLocalDataDirVal       = "/home/cdap/data"
```

Comment:
Adding a comment saying that operator assumes that a user "cdap" exists in the image for this to worker properly.

Reply:
Fixed by removing the hardcoded value.